docs/html/cli/pip_install.rst

```diff
@@ -137,16 +137,17 @@ step. This has two main practical benefits:
    an improvement.
 
 Although the new install order is not intended to replace (and does not replace)
-the use of ``setup_requires`` to declare build dependencies, it may help certain
-projects install from sdist (that might previously fail) that fit the following
-profile:
+the use of build dependencies, using either ``setup_requires`` (discouraged) and
+``build.requires``. The latter was proposed in `PEP_518`_ and will be the
+prefered way to specify build dependencies in the future.
+
+Using these methods may help certain projects install from sdist (that might
+previously fail) that fit the following profile:
 
 1. They have build dependencies that are also declared as install dependencies
    using ``install_requires``.
 2. ``python setup.py egg_info`` works without their build dependencies being
    installed.
-3. For whatever reason, they don't or won't declare their build dependencies using
-   ``setup_requires``.
```

Comment:
I removed this one after reading it 5 times. I still don't understand it.

Reply:
That whole paragraph (from "Although the new install order..." onwards) is pretty much obsolete, I think. It could probably just be removed altogether.

Reply:
Thanks, I was hoping someone would say that :smiley: